src/core/tracker.cpp

```diff
@@ -306,6 +306,9 @@ void Tracker::processPositionInformation( const GnssPositionInformation &positio
   if ( !mIsActive && !mIsReplaying )
     return;
 
+  if ( positionInformation.accuracyQuality() == GnssPositionInformation::AccuracyBad )
+    return;
```

Comment:
We'll need a Q_PROPERTY in the tracker.h 

bool filterAccuracy

to decide whether we filter this or not.